ramls/loan.json

```diff
@@ -242,6 +242,11 @@
       },
       "additionalProperties": false
     },
+    "itemEffectiveLocationIdAtCheckOut": {
+      "description": "The effective location, at the time of checkout, of the item loaned to the patron.",
+      "type": "string",
+      "$ref": "uuid.json"
+    },
```

Comment:
Is this needed by clients of the API, or is this property only intended for reporting?

Reply:
Reporting uses the public API to load its database.